src/plugins/nestedRows/data/dataManager.js

```diff
@@ -483,6 +483,7 @@ class DataManager {
    */
   addChildAtIndex(parent, index, element) {
     let childElement = element;
+    let flattenIndex;
```

Comment:
"Flatten" is a verb, so I'd probably go with `flattenedIndex` as a variable name here.

Reply:
Changed within https://github.com/handsontable/handsontable/pull/8713/commits/61c8e6e70eb29ee77e146927a59e5b0f6d321051.